scripts/linters/js_ts_linter.py

```diff
@@ -1117,6 +1117,49 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(eslint_output):
+        """Remove extra bits from eslint messages.
+
+        Args:
+            eslint_output: str. output returned by the eslint linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        error_messages = []
+        # Extract the message from list and split the message by newline
+        # so that we can use them and remove last four lines from the end.
+        # Becuase last two lines are empty strings and third one have a message
+        # with number of errors.
+        # Example: \u2716 2 problems (2 errors, 0 warnings)
+        # 1 error and 0 warnings potentially fixable with the `--fix` option.
+        eslint_output_lines = eslint_output.split('\n')
+        newlines_present = eslint_output_lines[-1] == '' and (
+            eslint_output_lines[-2] == '')
+        fix_option_present = eslint_output_lines[-3].endswith('`--fix` option.')
+        unicode_x_present = eslint_output_lines[-4].startswith('\u2716')
+
+        if (newlines_present and fix_option_present and unicode_x_present):
+            eslint_output_lines = eslint_output_lines[:-4]
+
+        for line in eslint_output_lines:
+            # ESlint messages start with line numbers and then a
+            # "x" and a message-id in the end. We are matching
+            # if the line contains line number because every message start with
+            # num:num where num is of type int and we are matching it with regex
+            # and if that is True then we are replacing "error" with empty
+            # string('') which is at the index 1 and message-id from the end.
+            if re.search(r'^\d+:\d+', line.lstrip()):
+                error_string = re.search(r'error', line).group(0)
+                message_id = re.search(r'(\w+-*)+$', line).group(0)
+                error_message = (
+                    line.replace(error_string, '').replace(message_id, ''))
```

Comment:
Same comment as above, to make sure just the key occurrence is replaced.

Reply:
Done.